### PR TITLE
github: add requesting review page

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/9162.

### CODE REVIEW

File: github/requesting_review.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Requesting review
+
+This document tires to explain who should be requested for review when crafting a PR.
```
Comment (rossf7) [inline github/requesting_review.md:3]:
```suggestion
This document tries to explain who should be requested for review when crafting a PR.
```

File: github/requesting_review.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Default
+
+By default a review **should** be requested from:
+
+- A team. Usually your team or [team owning the component][ownership].
```
Comment (rossf7) [inline github/requesting_review.md:9]:
```suggestion
- A team. Usually your team or the [team owning the component][ownership].
```

❤️ that we have the ownership page can link it.

File: github/requesting_review.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+requesting review on the whole change. Maybe their input is needed only for
+a part of the change and there is no reason to involve them in the whole PR.
+
+## Exceptions
+
+For information spreading:
+
+- Request review from relevant Chapters for documentation. E.g. Chapter PE for
+  documentation for creating releases.
+- Request review from relevant Chapters for `fmt`. E.g. Chapter FE for
```
Comment (ubergesundheit) [inline github/requesting_review.md:26]: I think for PRs in `happa` and `giantswarm-js-client` @giantswarm/chapter-front-end should always be pinged.

Comment (kopiczko) [reply github/requesting_review.md:26]: I'm fine with that. Can you suggest a change?

File: github/requesting_review.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+## Default
+
+By default a review **should** be requested from:
+
+- A team. Usually your team or the [team owning the component][ownership].
+- Individual stakeholders or person who can bring valuable input.
+
+By default a review **should not** be requested from:
+
+- Chapter or SIG.
```
Comment (ubergesundheit) [inline github/requesting_review.md:14]:
```suggestion
- Chapter or SIG. Exceptions are `happa` and `giantswarm-js-client`. PRs in these projects should always be reviewed by chapter FE.
```

File: github/requesting_review.md (hunk 3 above)
Comment (ubergesundheit) [reply github/requesting_review.md:26]: :point_up:

File: github/requesting_review.md (hunk 4 above)
Comment (kopiczko) [reply github/requesting_review.md:14]: Can we maybe add that to `Exceptions` section?

Comment (ubergesundheit) [reply github/requesting_review.md:14]: ok

File: github/requesting_review.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## Exceptions
+
+For information spreading:
+
+- Request review from relevant Chapters for documentation. E.g. Chapter PE for
+  documentation for creating releases.
+- Request review from relevant Chapters for `fmt`. E.g. Chapter FE for
+  JavaScript related guidelines.
+
```
Comment (ubergesundheit) [inline github/requesting_review.md:28]:
```suggestion
- PRs in projects `happa` and `giantswarm-js-client` should always be reviewed by chapter FE.

```

**Pull request merged**